pipelines/README.md

```diff
@@ -1,88 +1,90 @@
-- A target output for the data. Supported options are Apache Parquet files or a
-  FHIR server such as HAPI FHIR or
-  [Google Cloud Platform FHIR stores](https://cloud.google.com/healthcare/docs/how-tos/fhir).
-  - You can use our [HAPI FHIR server](#run-hapi-fhir-server-using-docker) image
-    for testing FHIR API targets.
-  - [Learn how to create a compatible GCP FHIR store](#create-a-google-cloud-fhir-store-and-bigquery-dataset),
-    if you want to use this option.
-- Build the [local version of Bunsen](bunsen/)
-
-### Installing the FHIR2 Module
-
-**To install via web**: Log in to OpenMRS as an administrator, go to **System
-Administration > Manage Modules > Search from Addons**, search for the `fhir2`
-module, and install. Make sure to install the `fhir2` module, not `fhir`.
-
-**To install via .omod file**: Go to
-[FHIR2 Module](https://addons.openmrs.org/show/org.openmrs.module.openmrs-fhir2-module)
-and download the .omod file. Go to **Advanced Administration > Module
-Properties** and find the modules folder. Move the .omod file to the modules
-folder then restart OpenMRS.
-
-[Learn more about OpenMRS modules](https://guide.openmrs.org/en/Configuration/customizing-openmrs-with-plug-in-modules.html).
-
-## Setup
-
-1.  [Install the FHIR2 Module](#installing-the-fhir2-module) if necessary.
-2.  [Clone](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
-    the
-    [OpenMRS FHIR Analytics](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics)
-    project to your machine.
-3.  Set the `utils` directory to world-readable: `chmod -r 755 ./utils`.
-4.  Build binaries with `mvn clean install`.
+## Pre-requisites
+
+**Note**: All commands are run from the git repository's root directory
+
+- Create an external Docker network named `cloudbuild` to run Docker images
+  listed below:
+  ```
+  docker network create cloudbuild
+  ```
+- Bring up FHIR source server. Either a:
+
+  - [HAPI source server and database](../docker/hapi-compose.yml) image. The
+    base URL for this server is `http://localhost:8081/fhir`. Run:
+    ```shell
+    docker-compose  -f ./docker/hapi-compose.yml up  --force-recreate -d
+    ```
+  - [OpenMRS Reference Application](../docker/openmrs-compose.yaml) image. The
+    base URL for this server is `http://localhost:8099/openmrs/ws/fhir2/R4`
+    ```shell
+    docker-compose -f ./docker/openmrs-compose.yml up --force-recreate -d
+    ```
+
+- Upload the Synthetic data stored in [sample_data](../synthea-hiv/sample_data)
+  to the FHIR server that you brought up using the
+  [Synthea Uploader](../synthea-hiv/README.md#Uploader). For example, to upload
+  to the HAPI FHIR server brought up in the previous step, run:
+  ```shell
+  python3 ./synthea-hiv/uploader/main.py HAPI http://localhost:8091/fhir \
+  --input_dir ./synthea-hiv/sample_data
+  ```
+- Bring up a target output for the data. Supported options are Apache Parquet
```

Comment:
We should make it clear that no extra config is needed if the output is only Parquet files (which is the main scenario). So please update to something like this:
```
- [optional] If the output is not only Apache Parquet files, and you want to output to
  a FHIR sink server like HAPI ...
```